pkgs/development/python-modules/pydeck/default.nix

```diff
@@ -1,11 +1,28 @@
-{ lib, buildPythonPackage, fetchPypi, ipykernel, ipywidgets, pythonOlder, pytestCheckHook, pandas }:
+{ lib
+, buildPythonPackage
+, fetchPypi
+, ipykernel
+, ipywidgets
+, pythonOlder
+, pytestCheckHook
+, pandas
+, jinja2
+, numpy
+, traitlets
+}:
 
 buildPythonPackage rec {
   pname = "pydeck";
   version = "0.7.1";
+  format = "setuptools";
 
   disabled = pythonOlder "3.7";
 
+  preBuild = ''
+    # unused
+    rm pyproject.toml
+  '';
```

Comment:
I don't see what would cause any harm in the file https://github.com/visgl/deck.gl/blob/master/bindings/pydeck/pyproject.toml

Reply:
It's because the pypi version's pyproject.toml violates the `[project]` key's schema. I'll add in the patch that fixes it.

Reply:
Fixed in #172055.